earthfile2llb/converter.go

```diff
@@ -1122,7 +1122,7 @@ func (c *Converter) ExpandArgs(word string) string {
 	return c.varCollection.Expand(word)
 }
 
-func (c *Converter) prepBuildTarget(ctx context.Context, fullTargetName string, platform *specs.Platform, allowPrivileged bool, buildArgs []string, isDangling bool, isFrom bool) (domain.Target, ConvertOpt, bool, error) {
+func (c *Converter) prepBuildTarget(ctx context.Context, fullTargetName string, platform *specs.Platform, allowPrivileged, doSaves bool, buildArgs []string, isDangling bool, isFrom bool) (domain.Target, ConvertOpt, bool, error) {
```

Comment:
I would merge `doSaves` and `isFrom` into a single parameter `cmdName` and use that.

```go
opt.doSaves = (cmdName == 'BUILD' && c.opt.doSaves)
```

Reply:
good plan, that made it much cleaner.